onc/+onc/OncDelivery.m

```diff
@@ -104,7 +104,7 @@
             %
             % Documentation: https://wiki.oceannetworks.ca/display/CLIBS/Data+product+download+methods
```

Comment:
Looks like this documentation link is old and confluence prompts a user to go here: https://wiki.oceannetworks.ca/display/O2A/Data+product+download+methods when it is used.

Reply:
Kan suggested to create another documentation page in matlab like other libraries do, and will abandon the one on wiki.oceannetworks.ca 
I will add the new page link once it's done